x = 10

def foo():
    print(x)                # error
    x = 20
    print(x)

foo()
